avaframe/com1DFA/com1DFA.py

```diff
@@ -1876,14 +1876,26 @@ def DFAIterate(cfg, particles, fields, dem, inputSimLines, simHash=""):
             log.debug(("cpu time Position = %s s" % (tCPU["timePos"] / nIter)))
             log.debug(("cpu time Neighbour = %s s" % (tCPU["timeNeigh"] / nIter)))
             log.debug(("cpu time Fields = %s s" % (tCPU["timeField"] / nIter)))
-            fieldsList, particlesList = appendFieldsParticles(
-                fieldsList, particlesList, particles, fields, resTypes
-            )
+
+            # Result parameters to be exported
+            if cfg["EXPORTS"].getboolean("exportData"):
+                # exportFields(cfg, Tsave, fieldsList, dem, outDir, cuSimName)
+                exportFields1(cfg, t, fields, dem, outDir, cuSimName, TSave="intermediate")
```

Comment:
Please modify the existing exportFields instead of creating a new copy of it. This way, the differences are transparent.

Reply:
in order to be loaded as timeseries in paraview - ascending numbers are required - hence it is just a counter for the files - it is set to zero outside the loop in line 1777 and from thereon one is added for each file saved initial in line 1787 and then inside the time iteration loop

Reply:
merged